index.ts

```diff
@@ -12,6 +11,48 @@ const autocompleteUrl = `${mapzenUrl}/autocomplete`
 const reverseUrl = `${mapzenUrl}/reverse`
 const searchUrl = `${mapzenUrl}/search`
 
+type Rect = {
+  maxLat: number
+  maxLon: number
+  minLat: number
+  minLon: number
+}
+type Circle = {
+  centerPoint: number
+  radius: number
+}
+type Boundary = {
+  circle?: Circle
+  country: string
+  rect: Rect
+}
+
+// apiKey is renamed depending on if it is being passed around or into fetch
+
+type Query = {
+  // Can be disabled as it is a hack to conform with api
+  // eslint-disable-next-line camelcase
+  api_key: string
+  apiKey?: string
+  boundary?: Boundary
+  focusPoint?: LonLatInput
+  format?: boolean
+  layers?: string
+  options?: RequestInit
```

Comment:
I'm guessing we'll see this a lot more as we continue adding typings, but perhaps it'd be helpful to note that this is a built-in typing.

Reply:
I added a note, but I may have misunderstood what you meant!